src/deadline/houdini_adaptor/HoudiniAdaptor/adaptor.py

```diff
@@ -255,34 +257,26 @@ def _handle_progress(self, match: re.Match) -> None:
         progress = int(percent)
         self.update_status(progress=progress)
 
-    def _handle_error(self, match: re.Match) -> None:
+    def _handle_license_error(self, match: re.Match) -> None:
         """
-        Callback for stdout that indicates an error or warning.
+        Callback for stdout that indicates a license error.
         Args:
             match (re.Match): The match object from the regex pattern that was matched in the
                               message
-
-        Raises:
-            RuntimeError: Always raises a runtime error to halt the adaptor.
         """
-        self._exc_info = RuntimeError(f"Houdini Encountered an Error: {match.group(0)}")
+        self._exc_info = RuntimeError(f"Houdini encountered a license error: {match.group(0)}")
 
-    def _handle_license_error(self, match: re.Match) -> None:
+    def _handle_error(self, match: re.Match) -> None:
         """
-        Callback for stdout that indicates an license error.
+        Callback for stdout that indicates an error or warning.
         Args:
-            match (re.Match): The match object from the regex pattern that was matched the message
+            match (re.Match): The match object from the regex pattern that was matched in the
+                              message
 
         Raises:
             RuntimeError: Always raises a runtime error to halt the adaptor.
         """
-        shutil_usage = shutil.disk_usage(os.getcwd())
-        self._exc_info = RuntimeError(
-            f"{match.group(0)}\n"
-            "This error is typically associated with a licensing error"
-            " when using Houdini. Check your licensing configuration.\n"
-            f"Free disc space: {shutil_usage.free//1024//1024}M\n"
-        )
```

Comment:
Just curious, why did we report the free disk space when we suspected license errors before?

Reply:
This was due to copy/pasting some hyper-specific maya license handling. They erroneously report licensing errors on the cloud by saying they ran out of disk space. So we did a few attempts at improving it to ensure that the user could easily verify that was not the case.